### ISSUE TITLE
Can vkCmdPipelineBarrier2KHR be used to perform queue family ownership transfers?

### ISSUE THREAD

**Issue by spinnylights** [KhronosGroup/Vulkan-Docs#1516]
The 1.2.177 spec says that `vkCmdPipelineBarrier` is used to define the release and acquire operations necessary for a queue family ownership transfer in [7.7.4. Queue Family Ownership Transfer](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#synchronization-queue-transfers). There is no mention of `vkCmdPipelineBarrier2KHR` in that section, and it states repeatedly that only `vkCmdPipelineBarrier` is suitable for this purpose, which would seem to imply that `vkCmdPipelineBarrier2KHR` is not.

However, the definitions of [`vkBufferMemoryBarrier2KHR`](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkBufferMemoryBarrier2KHR.html) and [`vkImageMemoryBarrier2KHR`](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkImageMemoryBarrier2KHR.html) state clearly that they can be used to define queue family ownership transfers. The only commands where these structures are used are in `synchronization2` commands for events and `vkCmdPipelineBarrier2KHR`; since the spec is clear in stating that events are not suitable for defining queue family ownership transfers, `vkCmdPipelineBarrier2KHR` would be the only means left to use them in this fashion. This would seem to imply that `vkCmdPipelineBarrier2KHR` _can_ be used to perform queue family ownership transfers.

Which is correct?

**Comment by Tobski** [comment]
Yes vkCmdPipelineBarrier2KHR is able to perform this function, that text was originally supposed to clarify that you can't use event set/wait to do a queue family ownership transfer, and I guess we just missed updating it  when we wrote sync2. I'll put together a fix for this.
